llama-index-integrations/vector_stores/llama-index-vector-stores-qdrant/llama_index/vector_stores/qdrant/base.py

```diff
@@ -574,7 +580,10 @@ def _create_collection(self, collection_name: str, vector_size: int) -> None:
                     # Newly created collection will have the new sparse vector name
                     sparse_vectors_config={
                         SPARSE_VECTOR_NAME: rest.SparseVectorParams(
-                            index=rest.SparseIndexParams()
+                            index=rest.SparseIndexParams(),
+                            modifier=rest.Modifier.IDF
+                            if "bm42" in self.fastembed_sparse_model
+                            else rest.Modifier.NONE,
```

Comment:
I recommend to parameterize the params during `__init__` ans not hard-code.

```py
rest.SparseVectorParams(**sparse_params)
```

Reply:
I think there needs to be some default though. I can add (yet another) init kwarg for this, that would override the default

Reply:
I think if users need that much customization though, they should create the collection outside of llama-index

Reply:
We get reports from people trying to create collections externally.
They run into issues by missing vector names or dimensions.

Reply:
Added optionally way to pass in and override default sparse and dense configs